release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -218,7 +223,7 @@ In the table, features are marked with the following statuses:
 |`lastTriggeredImageID` field in the `BuildConfig` spec for Builds
 |GA
 |DEP
-|
+|REM
```

Comment:
We aren't really removing it, we are just ignoring the field and not updating it.

Reply:
@adambkaplan, to avoid introducing `IGN` (Ignored) as a new stage in the deprecation/removal lifecycle, I've reworded the description. WDYT?

Reply:
Sounds good to me